src/pages/Result.vue

```diff
@@ -42,17 +58,22 @@ export default {
       }
     },
     computed: {
-      filteredList() {
+      filteredList() {          
+        if(this.loading == true) {
+          return {}
```

Comment:
This is just neatness but this returns an empty object but normally, the function returns a list